tests/test_unit/test_datamodules.py

```diff
@@ -98,6 +102,80 @@ def compare_transforms_attrs_excluding(transform1, transform2, keys_to_skip):
     return transform1_attrs_without_fns == transform2_attrs_without_fns
 
 
+@pytest.fixture()
+def create_dummy_dataset():
+    """Return a factory of dummy images and annotations for testing.
+
+    The created datasets consist of N images, with a random number of bounding boxes
+    per image. The bounding boxes have fixed width and height, but their location
+    is randomized. Both images and annotations are torch tensors.
+    """
+
+    def _create_dummy_dataset(n_images):
+        # n_images = 10  # needs to be > 5 to avoid floating point errors in dataset split
```

Comment:
Where this been assigned to 10? It is set to 5 for `test_collate_fn` and 50, 100 and 250 for `test_compute_splits`